File: csharp/examples/GetDocumentsBulkDownload.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net.Http;
namespace Examples.examples
{
    // Example of GET /documents/bulk-download

    public class GetDocumentsBulkDownload : BaseRequest
	{
        private string _documentIds;

        public GetDocumentsBulkDownload()
		{
            dotenv.net.DotEnv.Load();
            _documentIds = Environment.GetEnvironmentVariable("DOCUMENT_ID");
            Console.WriteLine($"GetDocumentsBulkDownload: _documentIds={_documentIds}");
        }

        public override string SendRequest()
        {
            // Construct the url with the ids param as an array in the query string
            var queryParams = new List<string>();
            foreach (var id in _documentIds.Split(','))
            {
                queryParams.Add($"ids[]={Uri.EscapeDataString(id)}");
            }
            var queryString = string.Join("&", queryParams);
            var urlWithParams = $"documents/bulk-download?{queryString}";
            return GetRequest(urlWithParams);
        }
    }
}

File: php/examples/get_documents_bulk_download.php
<?php
require_once 'base_request.php';

// Example of GET /documents/bulk-download

$debug = true;
$document_id   = $_ENV['DOCUMENT_ID'];
$request_data = '';
$request_params = ['ids' => [$document_id]];

send_request('GET', '/documents/bulk-download', $request_params, $request_data);
